docs/proposals/bulward-integration.md

```diff
@@ -67,6 +67,12 @@ spec:
 There are few reasons that we don't replace the `Account` with `Project`:
 - KubeCarrier cannot take fully advantage of Organization features in Bulward. For big enterprises, `Organzation` concept
 is needed for them to manage permissions of different team, different user has different permission for different project.
+- `Organization` is the top level in Bulward, and organization owners have freedom to manage Projects as they like, for example,
+moving objects around to `Project` namespaces to ease the management and resource isolation. While `Project` is not splittable,
+which means there is no sub-project in `Project`.
+- Services that available for the whole `Organization` can be shared by different `Project`s. Catalog related objects can
```

Comment:
This seems a bit messy. Thus you're allowing catalogs to live both in the organization as well as in the projects? What's the benefit in that over replicating it in each project. I'm viewing project as a closed sandbox where all project related stuff are contained.

Reply:
Nope, as I mentioned in the proposal, this is not a mandatory thing, and it depends on the organization owners.
1. Organization owners can keep all catalog stuff in the Organization namespace, and create `OrganizationRole` to mange members.
2. If Organization owners want to have more resource isolation, they can create Project, and move objects into Project namespace.

However, if we map Project to Account, we kinda lose the possibility of 1, i.e., Catalog cannot be shared in a Organization, and every Project will have a copy of catalog objects in namespace.

Reply:
how `Organizations` catalogs combined with `Projects` catalogs? I mean, if the project doesn't have any catalogs in its namespace, should we check organizations namespace for catalogs or this means that project don't have any catalogs?